bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -998,6 +1023,64 @@ private void updateChannelFavoritesList(List<Favorite> favorites) {
                 updateState(CHANNEL_FAVORITES_LIST, new StringType(favoritesList));
             }
         }
+
+        private void scheduleRequestFavorites() {
+            // Delay the execution to give the player thing handlers a chance to initialize
+            scheduler.schedule(() -> {
+                requestFavorites();
+            }, 3L, TimeUnit.SECONDS);
```

Comment:
```suggestion
            scheduler.schedule(this::requestFavorites, 3L, TimeUnit.SECONDS);
```

Reply:
I had actually tried to do it this way initially. Because the `requestFavorites` method is in `SqueezeBoxServerHandler`, not the inner class `SqueezeServerListener` it generates a compiler error. Is there a better way to handle this?

Reply:
Here's how you can do it.
```suggestion
            scheduler.schedule(SqueezBoxServerHandler.this::requestFavorites, 3L, TimeUnit.SECONDS);
```

Reply:
Ah, thanks.